var/spack/repos/builtin/packages/py-carputils/oc13.0_allow_ginkgo_flavor.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/carputils/settings/config.py b/carputils/settings/config.py
+index 7e25f47..f9b8925 100644
+--- a/carputils/settings/config.py
++++ b/carputils/settings/config.py
+@@ -292,6 +292,6 @@ def default_settings(software_root = None,
+     if flavor is None:
+         flavor = 'petsc'
+     else:
+-        assert flavor in ['pt', 'petsc', 'direct'], "Choose flavor among: 'pt', 'petsc' or 'direct'"
++        assert flavor in ['pt', 'petsc', 'direct', 'ginkgo'], "Choose flavor among: 'pt', 'petsc', 'direct' or 'ginkgo'"
```

Comment:
Is there a reason why we shouldn't make this change in the carputils repo?

Reply:
I would be happy if you could, I mostly wanted to collect all changes in this MR and do that relatively fast. If it's available by default in OpenCARP, then I would remove this patch here.

Reply:
Thanks! I removed the patch from `master` but keep it for `oc13.0` until there is a new release.

This also made me realize that I had forgotten to commit another part to this patch -- the resource installation was not done for the Ginkgo files.